src/Compiler/test/Microsoft.AspNetCore.Razor.Test.Common/WorkItemAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Roslyn.Test.Utilities;
+
+/// <summary>
+/// Used to tag test methods or types which are created for a given work item.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true)]
+public sealed class WorkItemAttribute : Attribute
```

Comment:
Feel free to move https://github.com/dotnet/razor/blob/main/src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/WorkItemAttribute.cs somewhere shared :)

Reply:
Oh, didn't see that one, thanks.

Reply:
Or, we can add a project that is shared among compiler and tooling tests.